Feature/data vault blog post

boxysean opened the pull request:
Resolves #3237 

## What are you changing in this pull request and why?

Adding some great Data Vault + dbt Cloud content.

## Checklist
- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

Commit:
Create blog post

Commit:
Create .gitkeep

boxysean committed:
Add images

boxysean committed:
Add Rasti image

Commit:
Add Rasti to authors

Commit:
Update image links

Commit:
Update description

boxysean commented:
@dbt-labs/product-docs I gave my best at making a pull request for this new blog post, but I'm sure I've messed something up.

I'm also looking for a content review before this hits the big time! Thanks!

Commit:
Fix authors

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+description: "When to use, and when not to use Data Vault 2.0 data modeling, and why dbt Cloud is a great choice"
+slug: data-vault-with-dbt-cloud
+
+authors: [rastislav_zdechovan], [sean_mcintyre]
```
fixing errors
```suggestion
authors: [rastislav_zdechovan] , sean_mcintyre]
```

mirnawong1 commented:
hey @boxysean , thanks for opening this up! I'll take a look at this for you ✨

mirnawong1 committed:
Merge branch 'current' into feature/data-vault-blog-post

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+Here’s an analogy to help illustrate Data Vault:
+
+Think of a city’s water supply. Each house does not have a pipe directly from the local river: there is a dam and a reservoir to collect water for the city from all of the sources – the lakes, streams, creeks, glaciers – before the water is redirected into each neighbourhood and finally into each home’s taps.
```
sugg to us spelling
```suggestion
Think of a city’s water supply. Each house does not have a pipe directly from the local river: there is a dam and a reservoir to collect water for the city from all of the sources – the lakes, streams, creeks, and glaciers – before the water is redirected into each neighborhood and finally into each home’s taps.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+A new development in the city? No problem! Just hook up the new pipes to the reservoir! Not enough water? Just find another water source and fill up the reservoir.
+
+Data Vault is the dam and reservoir: it is the well-engineered data model to structure an organisation’s data from source systems for use by downstream data projects – rather than each team collecting data straight from the source. The Data Vault data model is designed using a few well-applied principles, and in practice, pools source data so it is available for use by all downstream consumers. T ...
```
```suggestion
Data Vault is the dam and reservoir: it is the well-engineered data model to structure an organization’s data from source systems for use by downstream data projects – rather than each team collecting data straight from the source. The Data Vault data model is designed using a few well-applied principles, and in practice, pools source data so it is available for use by all downstream consumers. This promotes a scalable data warehouse through reusability and modularity.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+## Data Vault components
+
+Loading your data directly from source systems without applying any business rules implies that you want them stored in a so-called **Raw Vault**. This is most of the time the first step in the journey of transforming your data. There are situations where you’d want to apply business logic before loading the data into your presentation layer, that’s where **Business Vault **comes into play. Perfo ...
```
```suggestion
Loading your data directly from source systems without applying any business rules implies that you want them stored in a so-called **Raw Vault**. This is most of the time the first step in the journey of transforming your data. There are situations where you’d want to apply business logic before loading the data into your presentation layer, that’s where **Business Vault **comes into play. Performance enhancement or centralized business logic are a few of the reasons for doing so.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+Loading your data directly from source systems without applying any business rules implies that you want them stored in a so-called **Raw Vault**. This is most of the time the first step in the journey of transforming your data. There are situations where you’d want to apply business logic before loading the data into your presentation layer, that’s where **Business Vault **comes into play. Perfo ...
+
+The core components of Data Vault are hubs, links and satellites. They allow for more flexibility and extensibility, and can be used to model complex processes in an agile way.
```
```suggestion
The core components of Data Vault are hubs, links, and satellites. They allow for more flexibility and extensibility and can be used to model complex processes in an agile way.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+Here is what you need to know about the main components:
+
+* **Hubs**: A hub is the central repository of all business keys identifying the same business entity. By separating data into hubs, we ensure each piece of business concept is as accurate and consistent as possible, while avoiding redundancy and ensuring referential integrity;
```
```suggestion
* **Hubs**: A hub is the central repository of all business keys identifying the same business entity. By separating data into hubs, we ensure each piece of business concept is as accurate and consistent as possible, while avoiding redundancy and ensuring referential integrity;
```
```suggestion
* **Hubs**: A hub is the central repository of all business keys identifying the same business entity. By separating data into hubs, we ensure each piece of business concept is as accurate and consistent as possible while avoiding redundancy and ensuring referential integrity;
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+Here is what you need to know about the main components:
+
+* **Hubs**: A hub is the central repository of all business keys identifying the same business entity. By separating data into hubs, we ensure each piece of business concept is as accurate and consistent as possible, while avoiding redundancy and ensuring referential integrity;
+* **Links**: Links connect your hubs in Data Vault. The relationship is stored as data, which makes it auditable and flexible to change. There are several special types of links, but in most cases, links are bidirectional, meaning you can easily navigate back and forth between business entities. This allows you to analyse complex relationships via connections created by hubs and links in your dat ...
```
```suggestion
* **Links**: Links connect your hubs in Data Vault. The relationship is stored as data, which makes it auditable and flexible to change. There are several special types of links, but in most cases, links are bidirectional, meaning you can easily navigate back and forth between business entities. This allows you to analyze complex relationships via connections created by hubs and links in your data model;
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+* **Hubs**: A hub is the central repository of all business keys identifying the same business entity. By separating data into hubs, we ensure each piece of business concept is as accurate and consistent as possible, while avoiding redundancy and ensuring referential integrity;
+* **Links**: Links connect your hubs in Data Vault. The relationship is stored as data, which makes it auditable and flexible to change. There are several special types of links, but in most cases, links are bidirectional, meaning you can easily navigate back and forth between business entities. This allows you to analyse complex relationships via connections created by hubs and links in your dat ...
+* **Satellites**: Satellites store contextual, descriptive and historical information about the hubs and links they are attached to, depending on whether the data is related to a business object or a relationship. Each satellite in the Data Vault provides additional, valuable information about the main entity.
```
```suggestion
* **Satellites**: Satellites store contextual, descriptive, and historical information about the hubs and links they are attached to, depending on whether the data is related to a business object or a relationship. Each satellite in the Data Vault provides additional, valuable information about the main entity.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+To further accelerate the creation of these layers and prevent the repetition of the same business logic, you can make use of Business Vault as a complementary layer of your data warehouse.
+
+The Business Vault is there to fill the gaps of generic, source-data-generated Raw Vault, which often does not cover all of the business processes of your organisation. You can easily address such challenges by applying soft rules applied in this.
```
```suggestion
The Business Vault is there to fill the gaps of generic, source-data-generated Raw Vault, which often does not cover all of the business processes of your organization. You can easily address such challenges by applying soft rules applied in this.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+The Business Vault is there to fill the gaps of generic, source-data-generated Raw Vault, which often does not cover all of the business processes of your organisation. You can easily address such challenges by applying soft rules applied in this.
+
+Business Vault can also help with performance issues that can arise due to presentation layer transformations having to do lots of joins on the fly. In such scenarios, a business vault becomes a central piece for your business logic populating all of the information marts.
```
```suggestion
Business Vault can also help with performance issues that can arise due to presentation layer transformations having to do lots of joins on the fly. In such scenarios, a business vault becomes a central piece of your business logic populating all of the information marts.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+### Should you consider Data Vault for your data warehouse?
+
+Data Vault is powerful modelling technique for middle-to-enterprise level data warehouses with the following attributes:
```
```suggestion
Data Vault is a powerful modelling technique for middle-to-enterprise level data warehouses with the following attributes:
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+Don’t Repeat Yourself (DRY) software engineering principles can help you sleep better when you are dealing with complex projects, which Data Vault most often is.
+
+dbts’ **macros **feature is a life saver in terms of templating your code. It saves you headaches due to manual errors as well as defining transformation logic in one place in case you need to change it.
```
```suggestion
dbt's **macros **feature is a lifesaver in terms of templating your code. It saves you headaches due to manual errors as well as defining transformation logic in one place in case you need to change it.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+dbts’ **macros **feature is a life saver in terms of templating your code. It saves you headaches due to manual errors as well as defining transformation logic in one place in case you need to change it.
+
+Data Vault follows the insert only principle with incremental loading strategy. A built-in **jinja** functionality allows you to create one version of the dbt model for both incremental and full load of a table. The easy dependency management that this feature helps you achieve is a huge benefit for highly complex projects.
```
```suggestion
Data Vault follows the insert-only principle with incremental loading strategy. A built-in **Jinja** functionality allows you to create one version of the dbt model for both incremental and full load of a table. The easy dependency management that this feature helps you achieve is a huge benefit for highly complex projects.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+If you are new to the framework, taking a look at already built Data Vault macros can be crucial, and even if you are an expert, it can still be beneficial. dbt’s rich set of community **packages** can be directly applied to your project or used as an inspiration for your own transformation templates.
+
+Building your transformation templates leveraging reusable macros and flexible jinja language can enhance your project development in a scalable way. When things get more complex, you are able to go back and change your templates in one place either completely, or using parameters to ensure you don’t mess with what already works well.
```
```suggestion
Building your transformation templates leveraging reusable macros and flexible Jinja language can enhance your project development in a scalable way. When things get more complex, you are able to go back and change your templates in one place either completely, or using parameters to ensure you don’t mess with what already works well.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+Building your transformation templates leveraging reusable macros and flexible jinja language can enhance your project development in a scalable way. When things get more complex, you are able to go back and change your templates in one place either completely, or using parameters to ensure you don’t mess with what already works well.
+
+Scalability also happens at the database layer. With **materialisations**, you have fine-grained control over whether a database object built by dbt is persisted as a view, table, or built incrementally, which gives you control over the performance and cost characteristics of each transformation. So if your data platform bill is growing, it’s easy to identify which Data Vault components are the m ...
```
```suggestion
Scalability also happens at the database layer. With **materializations**, you have fine-grained control over whether a database object built by dbt is persisted as a view, table, or built incrementally, which gives you control over the performance and cost characteristics of each transformation. So if your data platform bill is growing, it’s easy to identify which Data Vault components are the most expensive and make optimizations to reduce cost.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+The biggest boon to Data Vault developer productivity in dbt Cloud are the **DataOps** and **Data Warehouse Automation** features of dbt Cloud. Each Data Vault developer gets their own development environment to work in and there is no complicated set up process to go through.
+
+Commit your work, create a penabled by **jobs** that can be defined for each environment separately and then scheduled or triggered by a specific event. Together with dbt **tags**, the feature allows you to orchestrate your project in an efficient and powerful way.
```
what do you mean by 'penabled'? do you mean 'create and enable **jobs**,  which you can define for each environment..

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+### Auditable data
+
+One of the main selling points of Data Vault is its auditability. In addition to its own capabilities, dbt Cloud features enhance this advantage even further. Each job execution leaves an **audit log**, which can be leveraged to analyse trends in job performance among other things, allowing you to identify bottlenecks in your system. dbt Cloud stores **artifact **files after each execution for fu ...
```
```suggestion
One of the main selling points of Data Vault is its auditability. In addition to its own capabilities, dbt Cloud features enhance this advantage even further. Each job execution leaves an **audit log**, which can be leveraged to analyze trends in job performance among other things, allowing you to identify bottlenecks in your system. dbt Cloud stores **artifact **files after each execution for further processing and analysis as well.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+One of the main selling points of Data Vault is its auditability. In addition to its own capabilities, dbt Cloud features enhance this advantage even further. Each job execution leaves an **audit log**, which can be leveraged to analyse trends in job performance among other things, allowing you to identify bottlenecks in your system. dbt Cloud stores **artifact **files after each execution for fu ...
+
+dbt has the built-in **data lineage **which helps both developers and data consumers understand just how the data assets in the data warehouse are created. And with the self-serve and automatically generated **dbt docs**, you can spend less time answering questions about your data from across the organisation and more time building your Data Vault.
```
```suggestion
dbt has the built-in **data lineage **which helps both developers and data consumers understand just how the data assets in the data warehouse are created. And with the self-serve and automatically generated **dbt docs**, you can spend less time answering questions about your data from across the organization and more time building your Data Vault.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+## How to get started with dbt Cloud and Data Vault
+
+There are many decisions to make before you roll up your sleeves and start implementing your Data Vault data warehouse. Apart from data modelling work, you need to agree on naming conventions, hash algorithm, staging strategy and data types for standard metadata attributes, and make sure these are all well documented. Here, to save yourself some headaches in the long run, we recommend starting yo ...
```
```suggestion
There are many decisions to make before you roll up your sleeves and start implementing your Data Vault data warehouse. Apart from data modelling work, you need to agree on naming conventions, hash algorithm, staging strategy, and data types for standard metadata attributes, and make sure these are all well documented. Here, to save yourself some headaches in the long run, we recommend starting your own **decision log**.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+There are many decisions to make before you roll up your sleeves and start implementing your Data Vault data warehouse. Apart from data modelling work, you need to agree on naming conventions, hash algorithm, staging strategy and data types for standard metadata attributes, and make sure these are all well documented. Here, to save yourself some headaches in the long run, we recommend starting yo ...
+
+In terms of the implementation of the Data Vault itself, we recommend familiarising yourself with the best practices well in advance, especially if you have no previous experience with the framework. There are two well known dbt packages focusing on Data Vault implementation, which you can take inspiration from to build your own templating system, or there can be used directly if they fit your us ...
```
```suggestion
In terms of the implementation of the Data Vault itself, we recommend familiarising yourself with the best practices well in advance, especially if you have no previous experience with the framework. There are two well-known dbt packages focusing on Data Vault implementation, which you can take inspiration from to build your own templating system, or there can be used directly if they fit your use case.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+### AutomateDV (formerly known as dbtvault)
+
+AutomateDV is the most popular open source Data Vault package for dbt, with some users having over 5000 Data Vault components in their project. Here in Infinite Lambda, we’ve been using this package for quite some time now, even building on top of it (depending on the specifics of the project). This mature system provides a great way to start your Data Vault with dbt Cloud journey as the learning ...
```
```suggestion
AutomateDV is the most popular open source Data Vault package for dbt, with some users having over 5000 Data Vault components in their project. Here in Infinite Lambda, we’ve been using this package for quite some time now, even building on top of it (depending on the specifics of the project). This mature system provides a great way to start your Data Vault with dbt Cloud journey as the learning curve is quite manageable, it is well documented and even comes with tutorials and working examples built on top of Snowflake’s TPCH standard dataset. There is one limitation to using the package and that is _AutomateDV _expects your source data to contain only one delta load. In order to work around this issue, owners of the package came up with custom dbt materializations to help you with the initial load of your system, however, the performance of such load is in our experience not acceptable. 
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+### datavault4dbt
+
+At first glance, this fairly new open source package works in a similar fashion, especially the usage of the macros provides the same experience (apart from names of some of the parameters). Diving deeper into documentation however, we can see it provides a higher level of customisation thanks to many global variables, which alters the behaviour of macros. It also supports any type of source data ...
```
```suggestion
At first glance, this fairly new open source package works in a similar fashion, especially since the usage of the macros provides the same experience (apart from the names of some of the parameters). Diving deeper into documentation, however, we can see it provides a higher level of customization thanks to many global variables, which alters the behavior of macros. It also supports any type of source data - CDC, transient or persistent, it can handle it all. We suggest looking into this package if you have a deeper understanding of Data Vault and need a complex, customizable system. It’s good to be aware of the fact that this package is new, so there is a risk of hidden unresolved issues. 
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+At first glance, this fairly new open source package works in a similar fashion, especially the usage of the macros provides the same experience (apart from names of some of the parameters). Diving deeper into documentation however, we can see it provides a higher level of customisation thanks to many global variables, which alters the behaviour of macros. It also supports any type of source data ...
+
+### Customising the existing packages
```
```suggestion
### Customizing the existing packages
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+### Customising the existing packages
+
+These two packages, AutomateDV and datavault4dbt, are the most popular approaches to building a Data Vault on dbt. However, sometimes these packages don’t quite match an organisation’s pre-existing Data Vault practices built with a different tool. At the surface, dbt looks quite simple, but deep down is extremely customisable: it’s possible to make minor modifications to the packages within your  ...
```
```suggestion
These two packages, AutomateDV, and datavault4dbt, are the most popular approaches to building a Data Vault on dbt. However, sometimes these packages don’t quite match an organization’s pre-existing Data Vault practices built with a different tool. At the surface, dbt looks quite simple, but deep down is extremely customizable: it’s possible to make minor modifications to the packages within your project using Jinja, which is a powerful templating language.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+These two packages, AutomateDV and datavault4dbt, are the most popular approaches to building a Data Vault on dbt. However, sometimes these packages don’t quite match an organisation’s pre-existing Data Vault practices built with a different tool. At the surface, dbt looks quite simple, but deep down is extremely customisable: it’s possible to make minor modifications to the packages within your  ...
+
+For example, some organisations choose different hashing algorithms to generate their Data Vault hash keys than what comes out-of-the-box with AutomateDV. So to change that, you can add a [dbt macro](https://docs.getdbt.com/docs/build/jinja-macros#macros) called [default__hash_alg_md5](https://github.com/Datavault-UK/automate-dv/blob/3db7cc285e110ae6976d0afe7a93adf9b776b449/macros/supporting/hash ...
```
```suggestion
For example, some organizations choose different hashing algorithms to generate their Data Vault hash keys than what comes out-of-the-box with AutomateDV. So to change that, you can add a [dbt macro](https://docs.getdbt.com/docs/build/jinja-macros#macros) called [default__hash_alg_md5](https://github.com/Datavault-UK/automate-dv/blob/3db7cc285e110ae6976d0afe7a93adf9b776b449/macros/supporting/hash_components/select_hash_alg.sql#L32C1-L36) to your project with the custom logic you want. Much of the package logic can be overridden in this way to suit your needs.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+### Build your own system
+
+Every project is different and needs its own set of features, special treatments tailored to your data or performance tuning mechanisms. Because of this, for any long term, high priority data warehouse solutions we at [Infinite Lambda](https://infinitelambda.com/) recommend working on your own templating system. It needs significant engineering effort before an actual implementation (and bug fixi ...
```
```suggestion
Every project is different and needs its own set of features, special treatments tailored to your data, or performance tuning mechanisms. Because of this, for any long term, high priority data warehouse solutions we at [Infinite Lambda](https://infinitelambda.com/) recommend working on your own templating system. It needs significant engineering effort before an actual implementation (and bug fixing during), but you’ll save time later thanks to knowing where to look for a potential issue. If you are not comfortable creating such a system from scratch, you can always start with one of the above open-source packages and build on them once you hit its limits.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+We at Infinite Lambda treat data quality very seriously and we push for high test coverage as well as overall data governance in every project. With the experience from multiple projects, we developed two data quality dbt packages, which can help business users raise trust in your data.
+
+Within the [dq_tools](https://hub.getdbt.com/infinitelambda/dq_tools/latest/) _package, we aim for simple storing test results and visualisation of these in a BI dashboard. Leveraging this tool can help with making sure your system behaves in an expected way, all in a visual format of dashboard built on your favourite BI tool. [dq_vault](https://hub.getdbt.com/infinitelambda/dq_vault/latest/) pac ...
```
```suggestion
Within the [dq_tools](https://hub.getdbt.com/infinitelambda/dq_tools/latest/) _package, we aim for simple storing test results and visualization of these in a BI dashboard. Leveraging this tool can help with making sure your system behaves in an expected way, all in a visual format of dashboard built on your favorite BI tool. [dq_vault](https://hub.getdbt.com/infinitelambda/dq_vault/latest/) package provides an overview of data quality for all Data Vault models in your dbt project. Complex as it is, Data Vault projects need detailed test coverage to make sure there are no holes in the system. This tool helps with governing your testing strategy and being able to identify issues very quickly.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+Within the [dq_tools](https://hub.getdbt.com/infinitelambda/dq_tools/latest/) _package, we aim for simple storing test results and visualisation of these in a BI dashboard. Leveraging this tool can help with making sure your system behaves in an expected way, all in a visual format of dashboard built on your favourite BI tool. [dq_vault](https://hub.getdbt.com/infinitelambda/dq_vault/latest/) pac ...
+
+To help you get started, we have created a template GitHub project you can utilise to understand basic principles of building Data Vault with dbt Cloud using one of the abovementioned packages. But if you need help building your Data Vault, get in touch.
```
```suggestion
To help you get started, we have created a template GitHub project you can utilize to understand the basic principles of building Data Vault with dbt Cloud using one of the abovementioned packages. But if you need help building your Data Vault, get in touch.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+### Entity Relation Diagrams (ERDs) and dbt
+
+There are a few open source packages to visualise the entities in your Data Vault built with dbt. I recommend checking out the [dbterd ](https://dbterd.datnguyen.de/1.2/index.html) which turns your [dbt relationship data quality checks](https://docs.getdbt.com/docs/build/tests#generic-tests) into an ER diagram.
```
```suggestion
There are a few open source packages to visualize the entities in your Data Vault built with dbt. I recommend checking out the [dbterd ](https://dbterd.datnguyen.de/1.2/index.html) which turns your [dbt relationship data quality checks](https://docs.getdbt.com/docs/build/tests#generic-tests) into an ER diagram.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+By leveraging the building blocks of Data Vault, organisations can build data warehouses that are adaptable to changing business requirements, promote data quality and integrity, and enable efficient data management and analytics. This in turn drives better decision-making, competitive advantage and business growth.
+
+Choosing the right methodology for building your data warehouse is crucial for your system’s capabilities in the long run. If you are exploring Data Vault and want to learn more, Infinite Lambda can help you make the right call for your organisation.
```
```suggestion
Choosing the right methodology for building your data warehouse is crucial for your system’s capabilities in the long run. If you are exploring Data Vault and want to learn more, Infinite Lambda can help you make the right call for your organization.
```

mirnawong1 commented on website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md:
```diff
@@ -0,0 +1,146 @@
+
+## Summary
+
+By leveraging the building blocks of Data Vault, organisations can build data warehouses that are adaptable to changing business requirements, promote data quality and integrity, and enable efficient data management and analytics. This in turn drives better decision-making, competitive advantage and business growth.
```
```suggestion
By leveraging the building blocks of Data Vault, organizations can build data warehouses that are adaptable to changing business requirements, promote data quality and integrity, and enable efficient data management and analytics. This in turn drives better decision-making, competitive advantage and business growth.
```

mirnawong1 commented:
hey @boxysean ! great article and thank you so much for opening this pr and doing such incredible configuration ✨ - I've left some grammatical feedback and changed it to us english per our style guide. i can also commit the suggs if you'd like! 

other than that, looks great!

mirnawong1 committed:
Merge branch 'current' into feature/data-vault-blog-post

Commit:
Update website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

Commit:
Update website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

Commit:
Update website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

Commit:
Update website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

Commit:
Update website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

Commit:
Update website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

Commit:
Mirna's language and grammar changes

Co-authored-by: mirnawong1 <89008547+mirnawong1@users.noreply.github.com>

mirnawong1 committed:
Update 2023-06-30-data-vault-2-0-with-dbt-cloud.md

add <!--truncate--> so the main blog page displays truncated content. currently its displaying entire blog

mirnawong1 commented:
this looks great to me and approving this for you @boxysean !  ✨

mirnawong1 committed:
Update 2023-06-30-data-vault-2-0-with-dbt-cloud.md

mirnawong1 reviewed (approved):
🚢

Commit:
Fix the "penabled" sentence

Commit:
Added links and changes suggested by Stephan

Commit:
Add link to demo project

Commit:
Update date, update tags

Commit:
Delete .gitkeep

Commit:
Fix tag

Commit:
Change image location

Commit:
Merge branch 'feature/data-vault-blog-post' of github.com:dbt-labs/docs.getdbt.com into feature/data-vault-blog-post

mirnawong1 committed:
Merge branch 'current' into feature/data-vault-blog-post
